pages/linux/ahost.md

```diff
@@ -0,0 +1,16 @@
+# ahost
+
+> DNS lookup utility to display the A or AAAA record linked with a hostname or IP address.
+> More information: <https://manned.org/ahost>.
+
+- Print an `A` or `AAAA` record associated with a hostname or IP address:
+
+`ahost {{example.com}}`
+
+- Display some extra debugging output:
+
+`ahost -d {{example.com}}`
+
+- Display the record with specified type:
+
+`ahost -t {{a|aaaa}} {{example.com}}`
```

Comment:
```suggestion
`ahost -t {{a|aaaa|u}} {{example.com}}`
```
Just a small suggestion to this page. 
>  If the type is "a", print the A record (default).  If the type is "aaaa",
              print the AAAA record.  If the type is "u", look for either AAAA or A
              record (in that order).
Ref: <https://manned.org/ahost>.